gnome/41-community-extras.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE wallpapers SYSTEM "gnome-wp-list.dtd">
+<wallpapers>
+    <wallpaper deleted="false">
+        <name>Eternal Ultramarine</name>
+        <filename>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Eternal-Ultramarine-Light.png</filename>
+        <filename-dark>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Eternal-Ultramarine-Dark.png</filename-dark>
+        <options>zoom</options>
+        <author>Inderix</author>
+    </wallpaper>
+    <wallpaper deleted="false">
+        <name>Lazuli</name>
+        <filename>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Lazuli-light.png</filename>
+        <filename-dark>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Lazuli-dark.png</filename-dark>
+        <options>zoom</options>
+        <author>Inderix</author>
+    </wallpaper>
+        <wallpaper deleted="false">
+        <name>Lumos Dawn</name>
+        <filename>/usr/share/backgrounds/ultramarine-linux/41-community-extras/Lumos-Dawn.PNG</filename>
+        <options>zoom</options>
+        <author>Artemis</author>
+    </wallpaper>
+    <wallpaper deleted="false">
+        <name>Really Long Name</name>
```

Comment:
is this really called "Really Long Name"?

Reply:
Yes